Automattic/newspack-plugin
# Remove material icons and rework Analytics and Ads Wizards

### All Submissions:






### Changes proposed in this Pull Request:

This PR entirely removes Material npm packages so I had to reorganise/rework some of the components (Plugin Installer, Waiting) and wizards (Analytics, Ads)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. See if you can find any Material icons left
3. Check the various wizards

### Other information:
